packages/web/src/utils/prerender.utils.tsx

```diff
@@ -0,0 +1,29 @@
+import { useMemo } from 'react'
+
+/* Web side prerender utils, to be used on the browser */
+
+export const isPrerendering = (): boolean => {
+  return !!global?.__REDWOOD_PRERENDER_MODE
```

Comment:
Maybe we can name this `__REDWOOD__` to keep it in line with the others?

Reply:
Oh sorry, didn't notice that +1

Reply:
Maybe something like `global.__REDWOOD_PRERENDER_MODE ?? false` is easier to read.

Reply:
Haha, sure. I'm not use the `??` syntax, but happy to change it.